src/Chullo.php

```diff
@@ -85,6 +86,31 @@ public function getResource($uri = "",
     }
 
     /**
+     * Gets a Fedora resoure's headers.
+     *
+     * @param string    $uri            Resource URI
+     * @param string    $transaction    Transaction id
+     *
+     * @return array    Headers of a resource.
+     */
+    public function getResourceHeaders($uri = "",
+                                       $transaction = "") {
+
+        // Ensure uri takes transaction into account.
+        $uri = $this->prepareUri($uri, $transaction);
+
+        // Send the request.
+        $response = $this->client->request(
+            'GET',
+            $uri
+        );
+
+        $code = $response->getStatusCode();
```

Comment:
So right now if something goes wrong, Guzzle throws its own ClientException.  I don't know if i want to mask that as some sort of FedoraRequestFailedException or some such.  For now, it's fine to leave as is.  We'll deal with the larger issue of 'how far do we abstract?' later.